src/bramble/client/ProjectLimiter.js

```diff
@@ -0,0 +1,94 @@
+define(function() {
+"use strict";
+    var ProjectLimiter = {};
+
+    var maxProjectState = {
+            fileSize: 0, // max size of any file in bytes. NOT USED now.
+            fileCount: 0, // total number of files you can have in a project
+            projectSize: 0// total number of bytes for all files in this project
+    };
+    var currentProjectState = {
+        numOfFiles: 0 ,
+        projectSize: 0
+    };
+
+    ProjectLimiter.initMaxLimits =  function (projectState){
+        maxProjectState = projectState;
+        console.log(projectState);
+    };
+   function _checkLimits(){
```

Comment:
See my comment below about needing a callback.  What you'll want to do is return an object on your callback that has all this data, and leave it up to the caller what to do (maybe `console.log` or maybe use the values to do things):

```js
ProjectLimiter.checkLimits = function(root, fs, oldSize, callback) {
    ....
    // eventually you send back your data.  I'm hard coding it here, but you'd have values you calculate:
    callback(null, { fileSize: 6, fileCount: 25, ... });
```